core/src/main/scala/com.snowplowanalytics.stream/loader/transformers/BadEventTransformer.scala

```diff
@@ -35,11 +39,10 @@ import Scalaz._
 /**
  * Class to convert bad events to ElasticsearchObjects
  *
- * @param the elasticsearch index name
- * @param the elasticsearch index type
+ * @param documentIndex elasticsearch index name
  */
-class BadEventTransformer(documentIndex: String, documentType: String)
```

Comment:
ideally, the `documentType` removal should have been in its own commit

Reply:
reverted.